packages/cli/src/ldap.ee/ldap.service.ee.ts

```diff
@@ -385,7 +401,8 @@ export class LdapService {
 		this.eventService.emit('ldap-general-sync-finished', {
 			type: !this.syncTimer ? 'scheduled' : `manual_${mode}`,
 			succeeded: true,
-			usersSynced: usersToCreate.length + usersToUpdate.length + usersToDisable.length,
+			usersSynced:
+				filteredUsersToCreate.length + filteredUsersToUpdate.length + usersToDisable.length,
 			error: errorMessage,
```

Comment:
I'm not sure, but in case some emails where invalid, should we save that somewhere (at least on the error) ?

Reply:
Yes, so far we only log this as a warning on the console, but we discussed it here as well that we should follow up with an improvement on the messaging. https://linear.app/n8n/issue/PAY-3180/provision-users-with-invalid-email-field I'm not sure where and how these notifications should look like.